plugins/BinarySearcher/DialogBinaryString.cpp

```diff
@@ -101,7 +93,10 @@ void DialogBinaryString::do_find() {
 						}
 					}
 
-					ui->progressBar->setValue(util::percentage(i, regions.size(), p - &pages[0], region->size()));
+					// update progress bar every 64KB
+					if (quint64(p) & 0xFFFF == 0) {
```

Comment:
Okay, thanks. I'll make another PR addressing this and some other pedantic warnings I've littered in my PRs.